admin/src/utils/prepareTrailFromSchema.js

```
@@ -0,0 +1,35 @@
+  ];
+
+  /**
+   * Walk the update object and create our trail
+   */
+
+  let trail = {};
+  let ignored = {};
+  
+  if (update && typeof update === 'object' && Object.keys(update).length > 0) {
+    Object.keys(update).forEach(key => {
+      if (schema?.attributes && schema.attributes.hasOwnProperty(key) && !ignoreProps.includes(key)) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)